partner-chains-cli/src/prepare_configuration/prepare_main_chain_config.rs

```diff
@@ -129,6 +132,21 @@ fn run_sidechain_main_cli_addresses<C: IOContext>(
 	Ok(())
 }
 
+fn prepare_native_token<C: IOContext>(context: &C) -> anyhow::Result<()> {
+	if context.prompt_yes_no("Do you want to configure a native token for you Partner Chain?", true)
+	{
+		NATIVE_TOKEN_POLICY.prompt_with_default_from_file_and_save(context);
+		NATIVE_TOKEN_ASSET_NAME.prompt_with_default_from_file_and_save(context);
+		ILLIQUID_SUPPLY_ADDRESS.prompt_with_default_from_file_and_save(context);
```

Comment:
`IlliquidCirculationSupplyValidator` is present in addresses output and should be placed in the part of JSON as the other addresses and policy ids.